app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
```

Comment:
Code Efficiency: The command is split twice here and in line 4. It would be more efficient to split it once and reuse the result.